SpotSwap/Supporting Files/AppDelegate.swift

```diff
@@ -18,7 +18,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         FirebaseApp.configure()
         //setup Window
 
-        if AuthenticationService.manager.getCurrentUser() != nil{
+        if let _ = AuthenticationService.manager.getCurrentUser(){
```

Comment:
I read in blogs thats its more expressive to write ` if AuthenticationService.manager.getCurrentUser() != nil ` when using if let to check for nil if you're also discarding the variable.